lib/dvl/simple_form_config.rb

```diff
@@ -48,7 +48,7 @@
   # Don't wrap checkboxes/radios *again*
   config.item_wrapper_tag = false
 
-  config.label_text = lambda { |label, required, _| "#{label} #{required}" }
+  config.label_text = lambda { |label, required, _| "#{label}&nbsp;#{required}".html_safe }
```

Comment:
I'm curious: this is to ensure that the text doesn't break in between a label and the `abbr`?